econ-article-feeder.py

```diff
@@ -15,22 +15,24 @@
 import sys
 
 
-def feed_econ(bucket_name, queue_name, rate, prefix, key_filter, working):
```

Comment:
This could maybe default to the most likely workflow that we use this for?
e.g. workflow_name="PublishPerfectArticle" as the final parameter